### PR TITLE
Create abishek.json

### PR DESCRIPTION
**Pull request by abishekgit**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3472"></a>

### CODE REVIEW

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]
Your file name must match your GitHub username, make it `data/abishekgit.json`

File: data/abishek.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Java Programmer, using my skills to create innovative solutions that solve not real-world but DSA problems.",
+  "avatar": "https://github.com/abishekgit.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/abishektwt",
+      "icon": "FaTwitter"
+    }
```
Comment (Aadarsh805) [inline data/abishek.json:12]: you are missing commas after each link object, fix those

File: data/abishek.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://instagram.com/abishekinsta",
+      "icon": "FaInstagram"
+    }
+    {
+      "name": "Follow me on Hashnode",
+      "url": "https://hashnode.com/@abishekydv",
+      "icon": "FaHashnode"
```
Comment (Aadarsh805) [inline data/abishek.json:21]: use this icon for Hashnode `SiHashnode`

File: data/abishek.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    }
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/abishek-in/",
+      "icon": "FaLinkedIn"
```
Comment (Aadarsh805) [inline data/abishek.json:26]: this should be `FaLinkedin`

File: data/abishek.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    }
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/abishekgit",
+      "icon": "FaGitHub"
```
Comment (Aadarsh805) [inline data/abishek.json:31]: and GitHub should be `FaGithub`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]
and your Hashnode link return 404

**Review by Tejender1521 — CHANGES_REQUESTED** [review]
Validate JSON using any online Json validator

**Comment by rahulpandey70** [comment]
Hey @abishekgit, It should be {your GitHub username}.json -> abishekgit.json

**Comment by SaraJaoude** [comment]
@abishekgit we are looking forward to having your Profile on LinkFree, but as it has been some time since you created this Pull Request and the suggested changes to have this merged have not been done, I will be closing it.  

I hope you will submit a new PR and you may wish to have a look at the QuickStart Guide in our docs which will cover those points made in the comments: https://linkfree.eddiehub.io/docs/quickstart

**Pull request closed without merging**